common/alerts.js

```diff
@@ -10,12 +10,15 @@
             if (alert.hasOwnProperty('type') && alert.hasOwnProperty('message')) {
                 return alerts.push(alert);
             }
-            console.log('Invalid alert properties: ', alert);
+            throw "Invalid alert";
```

Comment:
Don't throw an error. REmove if. add comment instead that says what properties are expected.

Create Alert class that takes in `type` and `message` args.